docs/guides/qiskit-code-assistant-jupyterlab.mdx

```diff
@@ -0,0 +1,133 @@
+</Admonition>
+
+
+<video title="A user uses the Qiskit Code Assistant in JupyterLab. The user has a comment in the code to get the available backends for their account in IBM Quantum, the Qiskit Code Assistant gives a code suggestion to get the available backends. The user executes the code, gets the available backends in IBM Quantum Platform for the current account. The user ask to the Qiskit Code Assistant to generate the code to get the T1, T2 and other properties for the qubit 1 in the device ibm_brisbane. The assistant gives a code suggestion. The user runs the suggested code and get the desired output." className="max-w-auto h-auto" controls>
+  <source src="/videos/guides/qiskit-code-assistant/demo-JupyterLab-t1t2-granite-8b-qiskit-short.mp4" type="video/mp4"></source>
+</video>
+
+## Install the JupyterLab extension
+To install the [JupyterLab extension](https://github.com/Qiskit/qiskit-code-assistant-jupyterlab), run the following command from a terminal:
+```sh
+pip install qiskit-code-assistant-jupyterlab
+```
+
+After installing the extension, start JupyterLab:
+
+```sh
+jupyter lab
+```
+
+The extension loads automatically and is listed on the bottom of the JupyterLab window.
+
+![Screenshot of the bar on the bottom of JupyterLab.](/images/guides/qiskit-code-assistant/jupyterlab-selected-model.png)
+
+For help working with JupyterLab, refer to the [JupyterLab documentation.](https://jupyterlab.readthedocs.io/en/latest/)
+
+## Configure extension settings
+
+It is recommended that you edit the following JupyterLab settings by going Settings > Settings Editor:
+
+- Click **Inline Completer**, find "Show widget" and choose **Always**.  This means that the the inline completer widget will always be shown so you can cycle through and select a completion item.
+
+- Click **Code Completion** and increase the value for "Default timeout for a provider." to `10000` or 10 seconds. The default value is 1 second, but the Qiskit Code Assistant API might take longer than this to find a suggstion. This setting only applies to the standard context menu that is invoked with `Tab`. The inline completer has a default of 10 seconds.
+
+Other settings you might want to change:
+
+- Keyboard shortcuts can be changed from Settings > Settings Editor > Keyboard Shortcuts.
+
+- You can change the IBM Quantum API token to use in the JupyterLab command palette. To do that, type `Alt + Shift + C`, search for `qiskit`, select the **Qiskit Code Assistant: Set IBM Quantum API token** command, and paste in your token.
+
+- [Advanced] To change the instance of the Qiskit Code Assistant Service that the extension should use, edit the Qiskit Code Assistant `serviceUrl` setting.
+
+- [Advanced] Keyboard shortcuts can be changed by searching for `completer` in the Keyboard Shortcuts settings (Settings > Settings Editor > Keyboard Shortcuts) and adding new shortcuts for the relevant commands.
```

Comment:
Feel free to disagree

```suggestion
- [Advanced] Keyboard shortcuts can be changed by searching for `completer` in the Keyboard Shortcuts settings (Settings -> Settings Editor -> Keyboard Shortcuts) and adding new shortcuts for the relevant commands.
```

Reply:
The dash is fine.  In my old tool, typing > added a little arrow, so this was just forgetfulness.